includes/compatibility/class-wcpdf-compatibility-third-party-plugins.php

```diff
@@ -16,9 +16,15 @@ function __construct()	{
 		if ( class_exists('WC_Subscriptions') ) {
 			if ( version_compare( \WC_Subscriptions::$version, '2.0', '<' ) ) {
 				add_action( 'woocommerce_subscriptions_renewal_order_created', array( $this, 'woocommerce_subscriptions_renewal_order_created' ), 10, 4 );
-			} else {
+			} 
+
+			if ( version_compare( \WC_Subscriptions::$version, '2.0', '>=' ) && version_compare( \WC_Subscriptions::$version, '2.5', '<' ) ) {
```

Comment:
This should be an `elseif` from the `if` above, like this:

```php
if ( version_compare( \WC_Subscriptions::$version, '2.0', '<' ) ) {
	add_action( 'woocommerce_subscriptions_renewal_order_created', array( $this, 'woocommerce_subscriptions_renewal_order_created' ), 10, 4 );
} elseif ( version_compare( \WC_Subscriptions::$version, '2.5', '<' ) ) {
	add_filter( 'wcs_renewal_order_meta', array( $this, 'wcs_renewal_order_meta' ), 10, 3 );
	add_filter( 'wcs_resubscribe_order_meta', array( $this, 'wcs_renewal_order_meta' ), 10, 3 );
} else {
	add_filter( 'wc_subscription_renewal_order_data', array( $this, 'wcs_renewal_order_meta' ), 10, 3 );
	add_filter( 'wc_subscriptions_resubscribe_order_data', array( $this, 'wcs_renewal_order_meta' ), 10, 3 );
}
```

Reply:
Yes, I'm sorry, you're right. 😢